executor/index_lookup_join.go

```diff
@@ -241,11 +257,15 @@ func (e *IndexLookUpJoin) lookUpMatchedInners(task *lookUpJoinTask, rowIdx int)
 	outerKey := task.encodedLookUpKeys.GetRow(rowIdx).GetBytes(0)
 	e.innerPtrBytes = task.lookupMap.Get(outerKey, e.innerPtrBytes[:0])
 	task.matchedInners = task.matchedInners[:0]
+
+	oldMemUsage := int64(cap(task.matchedInners)) * int64(unsafe.Sizeof(chunk.Row{}))
 	for _, b := range e.innerPtrBytes {
 		ptr := *(*chunk.RowPtr)(unsafe.Pointer(&b[0]))
 		matchedInner := task.innerResult.GetRow(ptr)
 		task.matchedInners = append(task.matchedInners, matchedInner)
 	}
+	newMemUsage := int64(cap(task.matchedInners)) * int64(unsafe.Sizeof(chunk.Row{}))
+	task.memTracker.Consume(newMemUsage - oldMemUsage)
```

Comment:
It's a frequent operation and the `matchedInners` takes a very small proportion of the memory of the task.
How about just ignore it?